Shared/Samples/Edit feature attachments/EditFeatureAttachmentsView.swift

```diff
@@ -0,0 +1,233 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct EditFeatureAttachmentsView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The location that the user tapped on the map.
+    @State private var screenPoint: CGPoint?
+    /// A Boolean value indicating whether the features are loaded.
+    @State private var isLoaded = false
```

Comment:
Per my test, it takes less than 1 second for features to appear, so I would say don't add the loading overlay for this sample.